audit/projects/k8s-artifacts-prod/services/enabled.txt

```diff
@@ -1,6 +1,5 @@
 NAME                                 TITLE
 bigquery.googleapis.com              BigQuery API
-bigquery.googleapis.com              BigQuery API
```

Comment:
Not sure why we had a dupe entry here to begin with, not sure what caused it to get removed